code/modules/mob/living/simple_animal/hostile/illusion.dm

```diff
@@ -58,7 +58,7 @@
 			return
 		var/mob/living/simple_animal/hostile/illusion/M = new(loc)
 		M.faction = faction.Copy()
-		M.Copy_Parent(parent_mob, life_span, health/2, melee_damage_upper, multiply_chance)
+		M.Copy_Parent(parent_mob, 80, health/2, melee_damage_upper, multiply_chance)
```

Comment:
Couldn't you do `life_span-world.time`, which would result in all the clones disappearing at about the same time?